Plots/MapProjections/NCL_native_2.py

```diff
@@ -0,0 +1,91 @@
+"""
+NCL_native_2_lg.py
+================
+
+This script illustrates the following concepts:
+   - Drawing filled contours over a mercator map
+   - Overlaying contours on a map without having lat,lon coordinates
+   - Drawing a map using the medium resolution map outlines
+   - Turning on map tickmark labels with degree symbols
+   - Selecting a different color map
+   - Zooming in on a particular area on a mercator map
+   - Using best practices when choosing plot color scheme to accomodate visual impairments
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/native_2.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/native_2_lg.png
+"""
+
+###############################################################################
+
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+import cartopy.feature as cfeature
```

Comment:
To keep things consistent with other examples, could you move the cartopy imports next to each other